urbanairship-react-native/js/UrbanAirship.ts

```diff
@@ -639,13 +639,21 @@ export class UrbanAirship {
   /**
    * Removes a listener for an Urban Airship event.
    *
+   * @param listener The event listener (an emitter subscription).
+   */
+  static removeListener(listener: EmitterSubscription) {
+    EventEmitter.removeSubscription(listener);
```

Comment:
According to the [stack overflow question](https://stackoverflow.com/questions/36886628/how-do-you-remove-a-listener-from-react-natives-eventemitter-instance/36901160) you referenced, the listener is removed by calling `remove()` on the subscription, or in this case:
```
listener.remove();
```
It looks like both ways work, but I'm curious why did you not do it that way?

Reply:
`removeSubscription` in UAEventEmitter had some additional code in it, I figured that was more important in terms of Airship.

```
removeSubscription(subscription) {
    if (_reactNative.Platform.OS === 'android') {
      UrbanAirshipModule.removeAndroidListeners(1);
    }

    super.removeSubscription(subscription);
  }
```

Reply:
Ah. Good catch. I didn't realize `EventEmitter` was a `UAEventEmitter`.